rsconf/component/sirepo.py

```diff
@@ -185,14 +186,13 @@ def _install_dirs_and_files(self):
             self.install_directory(p.join(c))
         self.install_access(mode='400')
         for c in z.feature_config.proprietary_sim_types:
-            if c != 'jupyterhublogin':
-                self.install_abspath(
-                    self.proprietary_file(jc, c),
-                    p.join(c, c + '.tar.gz'),
-                )
+            self.install_abspath(
+                self.proprietary_file(jc, c),
+                p.join(c, c + '.tar.gz'),
+            )
 
     def _jupyterhublogin(self, z):
-        z.jupyterhub_enabled =  'jupyterhublogin' in self.j2_ctx.sirepo.feature_config.proprietary_sim_types
+        z.jupyterhub_enabled =  'jupyterhublogin' in self.j2_ctx.sirepo.feature_config.default_proprietary_sim_types
```

Comment:
i think this needs to be proprietary_sim_types here, too, because it is a proprietary_sim_type and it is a default_proprietary_sim_type. I think you could add() the two sets so you ensure that proprietary_sim_types is set properly in feature_config as well as default_proprietary_sim_types.

Reply:
Can you elaborate? Why is it both types? In [sirepo.feature_config we assert](https://github.com/radiasoft/sirepo/blob/master/sirepo/feature_config.py#L122) that it is in one or the other. It is nice to have them separate in rsconf because then we only do the proprietary file stuff for codes in proprieatary_sim_types.

Reply:
alrighty that makes sense so ignore my comment.